frontend/simple/assets/sass/_groupincome.scss

```diff
@@ -25,6 +25,25 @@ body {
   }
 }
 
+a {
+  border-bottom: 1px solid $link;
+  transition: padding-bottom ease-out .05s, color ease-out .15s;
+
+  &:hover {
```

Comment:
Hi Pieer 👋 
This is a direct modification to `a` from Bulma's styles, so the right place for this override to be done is at `bulma_overrides/base/generic.scss`.

The way to know exactly that's the place is just by checking what's the source of the original styles.
When you inspect a link you see the styles coming from `generic.sass`. After checking the complete path you notice `bulma/sass/base/generic`. So our overrides structure should follow the same for easier debugger and overrides.

This file `_groupincome.scss` should be used only for new global classes.

Does this make sense to you?

Reply:
Hi Sandrina,
I start to understand what you did here. 
I was wondering why everything seems to be under a Bulma override folder.
I'm not a big fan of css framework I have to say.. overriding is not my cup of 🍵 

In that case you consider that the link tag is part of bulma framework?
-> Ok, not my point of view but I understand the logic.

Thanks for reviewing and sharing your architecture strategy.

Reply:
It does make sense to me as well that if Bulma is defining the default style of the links we are using, that to override that style we should be clear about that fact by have a Bulma overrides folder. Otherwise it's not clear what is being modified.

> I'm not a big fan of css framework I have to say

But your application used [Buefy](https://buefy.github.io), which seems like a pretty great CSS/components framework to me. :D

(I'm actually wondering if we should use some of the components from that in our app. I wouldn't mind, as the code seems well written!)

Reply:
My app didn't include Buefy no. I've only copy one component logic that I've simplified :P
I understand the logic however. And I will follow your recommandation! :D